integration

```diff
@@ -138,6 +138,7 @@
   "briis/securityspy",
   "briis/weatherbit",
   "briis/weatherflow_forecast",
+  "brmccrary/nuvo_tuner",
```

Comment:
The current release has an invalid file <https://github.com/brmccrary/nuvo_tuner/blob/v1.0/hacs.json>